infra-as-code/bicep/modules/hubNetworking/README.md

```diff
@@ -25,7 +25,7 @@ The module requires the following inputs:
  | parPrivateDnsZones            | array  | See example parameters file [`hubNetworking.parameters.json`](hubNetworking.parameters.example.json) | Array of DNS Zones to provision in Hub Virtual Network. Default: All known Azure Private DNS Zones except for: `privatelink.batch.azure.com`, `privatelink.azmk8s.io` and `privatelink.siterecovery.windowsazure.com` as these are region specific, which you can add to the parameters file with the required region in the zone name that you wish to deploy for. For more details on private DNS Zones please refer to the above link. | None                          | See Default                  |
  | parCompanyPrefix              | string | alz                                                                                                  | Prefix value which will be pre-appended to all resource names                                                                                                                                                                                                                                                                                                                                                                             | 1-10 char                     | alz                          |
- | parDdosPlanName               | string | ${parCompanyPrefix}-DDos-Plan                                                                        | Name which will be associated with distributed denial of service protection plan                                                                                                                                                                                                                                                                                                                                                          | 1-80 char                     | alz-DDos-Plan                |
+ | parDdosPlanName               | string | ${parCompanyPrefix}-Ddos-Plan                                                                        | Name which will be associated with distributed denial of service protection plan                                                                                                                                                                                                                                                                                                                                                          | 1-80 char                     | alz-Ddos-Plan                |
```

Comment:
```suggestion
 | parDdosPlanName               | string | ${parCompanyPrefix}-ddos-plan                                                                        | Name which will be associated with distributed denial of service protection plan                                                                                                                                                                                                                                                                                                                                                          | 1-80 char                     | alz-ddos-plan                |
```

Reply:
Make all actual resources names lowercase 👍

Reply:
can you also update the example output value for `outDdosPlanResourceID` to match this change also 👍

Reply:
> can you also update the example output value for `outDdosPlanResourceID` to match this change also 👍

Done, had to change alz-ddos-plan on multiple places (doc, test, example.json and bicep file...). Doing a deployment before submitting new change.